index.html

```diff
@@ -1287,37 +1303,41 @@ <h4>
   <section> <h3>Push a smart poster message</h3>
     <pre class="example">
       const writer = new NDEFWriter();
+      const encoder = new TextEncoder();
       writer.push({ records: [
         {
-          recordType: "smart-poster",
+          recordType: "smart-poster",  // Sp
           data: { records: [
             {
-              recordType: "url",
-              data: "https://my.org/content/19911"
+              recordType: "url",  // URL record for the Sp content
+              data: encoder.encode("https://my.org/content/19911")
```

Comment:
For "url" and "text", you can use strings directly.
See https://w3c.github.io/web-nfc/#push-a-url-to-either-a-tag-or-peer